README.Rmd

```diff
@@ -16,10 +16,10 @@ knitr::opts_chunk$set(
 # goodpractice <img src="man/figures/logo.png" align="right" width="20%" height="20%" />
 
   <!-- badges: start -->
-  [![R-CMD-check](https://github.com/MangoTheCat/goodpractice/workflows/R-CMD-check/badge.svg)](https://github.com/MangoTheCat/goodpractice/actions)
+  [![R-CMD-check](https://github.com/ropensci-review-tools/goodpractice/workflows/R-CMD-check/badge.svg)](https://github.com/ropensci-review-tools/goodpractice/actions)
   [![CRAN status](https://www.r-pkg.org/badges/version/goodpractice)](https://CRAN.R-project.org/package=goodpractice)
   [![CRAN RStudio mirror downloads](https://cranlogs.r-pkg.org/badges/goodpractice)](https://www.r-pkg.org/pkg/goodpractice)
-  [![Codecov test coverage](https://codecov.io/gh/MangoTheCat/goodpractice/branch/main/graph/badge.svg)](https://app.codecov.io/gh/MangoTheCat/goodpractice?branch=main)
+  [![Codecov test coverage](https://codecov.io/gh/ropensci-review-tools/goodpractice/branch/main/graph/badge.svg)](https://app.codecov.io/gh/ropensci-review-tools/goodpractice?branch=main)
```

Comment:
occasion to add an R-universe badge?

Reply:
Should really do that to all packages, so via a separate issue. I'd also like to keep this PR as minimal as possible.